buildspec.yml

```diff
@@ -0,0 +1,69 @@
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - apk update
+      - apk add python3
+      - apk add python3-dev
+      - apk add py3-setuptools
+      - pip3 install awscli
+      - python3 --version
+      - terraform --version
+      - START_DIR=$(pwd)
+      - cd $START_DIR/modules/rdcb
+      - aws s3 cp s3://$AWS_S3_BUCKET/modules/rdcb/terraform.tfvars .
+      - terraform init -no-color -input=false
+      - cd $START_DIR/modules/rdsh
+      - aws s3 cp s3://$AWS_S3_BUCKET/modules/rdsh/terraform.tfvars .
+      - terraform init -no-color -input=false
+      - cd $START_DIR/modules/rdgw
+      - aws s3 cp s3://$AWS_S3_BUCKET/modules/rdgw/terraform.tfvars .
+      - terraform init -no-color -input=false
+      - cd $START_DIR/modules/guac
+      - aws s3 cp s3://$AWS_S3_BUCKET/modules/guac/terraform.tfvars .
+      - terraform init -no-color -input=false
+  build:
+    commands:
+      - cd $START_DIR/modules/rdcb
+      - terraform plan -no-color -out=tfplan -input=false
+      - terraform apply -no-color -input=false tfplan
+      - MY_VAR=$(terraform output rdcb-hostname -no-color)
```

Comment:
can you make this var name more descriptive please?

Reply:
Will do